src/mutation_data.cpp

```diff
@@ -104,6 +104,7 @@ void mutation_category_trait::load( const JsonObject &jsobj )
 
     jsobj.get_member( "mutagen_message" ).read( new_category.raw_mutagen_message );
     new_category.wip = jsobj.get_bool( "wip", false );
+    new_category.skip_test = jsobj.get_bool( "skip_test", false );
```

Comment:
Looks like `skip_test` currently serves the same purpose as `wip`. Any reason not to just use `wip` instead?

Reply:
Nomenclature, mainly - `wip` is misleading for categories that aren't still in the process of being developed, so I wanted to avoid having a misnomer. I can merge the two into a unified `skip_test` variable if that'd work, although I'm not sure if I should put in obsoletion warnings and etc to account for mods that might use it.